src/artemis/devices/DCM.py

```diff
@@ -0,0 +1,24 @@
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsMotor, EpicsSignalRO
+
+
+class DCM(Device):
+
+    # upstream_x: EpicsMotor = Cpt(EpicsMotor, '-MO-DET-01:UPSTREAMX')
+
```

Comment:
Nit: Is this used here? We should remove it if not

Reply:
no, that comment has sneaked in from elsewhere. Will remove